app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupAutoFillViewModel.kt

```diff
@@ -72,11 +72,12 @@ class SetupAutoFillViewModel @Inject constructor(
     }
 
     private fun handleTurnOnLaterConfirmClick() {
-        sendEvent(SetupAutoFillEvent.NavigateToCompleteSetup)
+        // TODO PM-10631 record user chose to turn on later for settings badging.
+        // TODO PM-10632 update status to complete setup step.
```

Comment:
So we ditched the event and now we just need to update the state and then state-based nav just does the navigation for us?

Reply:
yes! We will need to add another onboarding status for "FINISH_SETUP" for navigating to the final confirm screen, but yeah none of the onboarding will need event navigation. When we complete the user state will update then the root nav state will be "VaultUnlocked"